composer/Dockerfile

```diff
@@ -53,8 +51,7 @@ RUN mkdir /tmp/composer-cache \
   && rm -rf /tmp/composer-cache
 
 COPY --chown=dependabot:dependabot composer/helpers /opt/composer/helpers
-RUN bash /opt/composer/helpers/v1/build \
-  && bash /opt/composer/helpers/v2/build
+RUN bash /opt/composer/helpers/build
```

Comment:
@Seldaek @stof Is this a reasonable assumption? 

Because if I'm wrong and you expect to ship `composer` `v3` with breaking changes in the not too distant future, then I'll keep the version-selection logic around...